external-crates/move/crates/move-analyzer/trace-debug/src/extension.ts

```diff
@@ -353,18 +352,26 @@ class MoveConfigurationProvider implements vscode.DebugConfigurationProvider {
 }
 
 /**
- * Finds the trace information for the current active editor.
+ * Finds the trace information for the current active editor. If the trace
```

Comment:
Finding trace info containing a single traced function only makes sense for unit tests where we indeed trace a single function. For traces generated by the replay tool this does not hold in general